packages/services/src/workers/monitor-keeper.js

```diff
@@ -1,88 +1,90 @@
-import postmark from 'postmark'
+import { ServerClient } from 'postmark'
+import Etherscan from '../models/Etherscan'
 import Models from '@aragonone/court-backend-server/build/models'
 import Network from '@aragonone/court-backend-server/build/web3/Network'
 
-// notifications settings
-const FROM = 'noreply@aragon.one'
+import { fromWei } from 'web3-utils'
+import { bigExp } from '@aragonone/court-backend-shared/helpers/numbers'
+import getWalletFromPk from '@aragonone/court-backend-shared/helpers/get-wallet-from-pk'
 
-// number of blocks checked backwards first time
-const BLOCKS_BACKWARDS_INITIAL_THRESHOLD = 10
+const FROM = 'noreply@aragon.one'
+const BALANCE_THRESHOLD = bigExp(1, 17) // 0.1 ETH
 
-let lastCheckedBlockNumber = 0
+const { Admin, KeeperSuspiciousTransaction } = Models
 
 export default async function (logger) {
-  const court = await Network.getCourt()
-  const { environment } = court
-  const keeper = await getKeeperAddress(environment)
-  const courtAddresses = await getWhitelistedAddresses(court)
+  const { environment: { network } } = Network
+  const etherscan = new Etherscan(network, process.env.ETHERSCAN_API_KEY)
+  const keeper = getWalletFromPk(process.env.PRIVATE_KEY).getAddressString()
 
-  if (lastCheckedBlockNumber === 0) {
-    lastCheckedBlockNumber = await environment.getLastBlockNumber() - BLOCKS_BACKWARDS_INITIAL_THRESHOLD
-  }
-  lastCheckedBlockNumber = await monitor(logger, environment, courtAddresses, keeper, lastCheckedBlockNumber)
+  await monitorTransactions(logger, etherscan, keeper, network)
+  await monitorEthBalance(logger, etherscan, keeper, network)
 }
 
-async function monitor(logger, environment, courtAddresses, keeper, lastCheckedBlockNumber) {
+async function monitorTransactions(logger, etherscan, keeper, network) {
+  const court = await Network.getCourt()
+  const courtAddresses = await getWhitelistedAddresses(court)
+  const lastInspectedBlockNumber = (await KeeperSuspiciousTransaction.lastInspectedBlockNumber()) + 1
+  logger.info(`Checking transactions for keeper address ${keeper} from block ${lastInspectedBlockNumber}`)
+  const transactions = await etherscan.getTransactionsFrom(keeper, lastInspectedBlockNumber)
+  if (transactions.length === 0) return logger.info('No transactions found')
+
   try {
-    const currentBlockNumber = await environment.getLastBlockNumber()
-    logger.info(`Checking transactions for keeper address ${keeper} from block ${lastCheckedBlockNumber + 1} to ${currentBlockNumber}`)
-    for (let blockNumber = lastCheckedBlockNumber + 1; blockNumber <= currentBlockNumber; blockNumber++) {
-      await checkTransactions(logger, environment, courtAddresses, blockNumber, keeper)
+    for (const transaction of transactions) {
+      const { hash, to, value, blockNumber } = transaction
+      logger.info(`Found transaction ${hash} on block ${blockNumber}`)
```

Comment:
For the logging, maybe it would make sense to put this in an else clause after the if, so we avoid seeing both this and then the "suspicious" entry in the logs.